### PR TITLE
Task #54436 - Discuss card UI changes

### PR DESCRIPTION
**Pull request by SuvajitTarento**
Changes for discuss UI in the home page

### CODE REVIEW

File: library/ws-widget/collection/src/lib/card-home-discuss/card-home-discuss.component.scss [hunk 1]
```diff
@@ -152,4 +169,27 @@
 
 .mat-card-footer {
   padding-bottom: 22px;
+  margin-left: -18px;
+}
+
+.mat-card-sub-title-marign
```
Comment (christyfernandes) [inline library/ws-widget/collection/src/lib/card-home-discuss/card-home-discuss.component.scss:175]: Dont override the common classes and styles

File: library/ws-widget/collection/src/lib/card-home-discuss/card-home-discuss.component.scss [hunk 2]
```diff
@@ -152,4 +169,27 @@
...
+{
+  margin-left: -18%;
```
Comment (christyfernandes) [inline library/ws-widget/collection/src/lib/card-home-discuss/card-home-discuss.component.scss:177]: Why is the need for minus margins. can we avoid this and fix

File: library/ws-widget/collection/src/lib/card-home-discuss/card-home-discuss.component.scss (hunk 1 above)
Comment (christyfernandes) [reply library/ws-widget/collection/src/lib/card-home-discuss/card-home-discuss.component.scss:175]: If u have to override then use one more custom class name and then give additional styles

**Review by christyfernandes — CHANGES_REQUESTED** [review]

File: library/ws-widget/collection/src/lib/card-home-discuss/card-home-discuss.component.scss (hunk 2 above)
Comment (SuvajitTarento) [reply library/ws-widget/collection/src/lib/card-home-discuss/card-home-discuss.component.scss:181]: I tried to remove it.. but to align it i need to add this margin left . i didn't overwrite the main class as you suggested.. i have created a custom class for that

**Review by christyfernandes — APPROVED** [review]

**Pull request merged**